.github/workflows/benchmarks.yml

```diff
@@ -8,16 +8,12 @@ jobs:
   benchmarks:
     name: "Run benchmarks"
 
-    runs-on: ${{ matrix.operating-system }}
+    runs-on: "ubuntu-latest"
 
     strategy:
       matrix:
-        dependencies:
-          - "locked"
         php-version:
           - "7.4"
-        operating-system:
-          - "ubuntu-latest"
```

Comment:
I surely miss something, but I don't see how it can ease some kind of management since all action files are using the same `runs-on`.

Reply:
Considering the OS as a variable helps adding perhaps Windows in the future, as well as keep in sync with template repositories. I can't tell you if @lcobucci is doing this, but I do and it's easier to just diff the `strategy.matrix` section and leave everything else copy-pasted across repositories.

Reply:
At the beginning, I remove this variable since it was always the same and used only for `runs-on`. But your story about templates and keeping things in sync convinces me. I'm going to push modifications.
Thanks for the advice 👍